libs/blocks/merch-card/merch-card.css

```diff
@@ -238,6 +238,34 @@
   border-color: var(--color-accent);
 }
 
+.merch-card.has-divider hr {
+  margin: var(--spacing-xxs) 0;
+}
+
+.merch-card.has-divider .consonant-CardFooter hr {
+  display: none;
+}
+
+.merch-card.background-opacity-70 {
+  background-color: rgba(255 255 255 / 70%);
```

Comment:
could this be a variable as well?

Reply:
@yesil - This definitely could be a var. I'm open, but I think till we have a use case for other blocks to use this i'll likely keep it as is for now.